src/client/render/cameras.js

```diff
@@ -288,6 +305,9 @@ function TrackingCamera(
 			75, renderer.domElement.clientWidth / renderer.domElement.clientHeight, 0.1, 5000
 		);
 
+	if (!options.enabledByDefault)
```

Comment:
This check would set `enabledByDefault` to true if the option is explicitly set to false.

Reply:
Heck, how did I manage to do this for one of the cameras but not the other